tests/core/gadget-config-defaults-to-snaps/task.yaml

```diff
@@ -100,6 +110,9 @@ restore: |
     . "$TESTSLIB"/systemd.sh
 
     SUFFIX="$(get_test_snap_suffix)"
+    
+    # remove snap with default-configure hook
+    snap remove --purge "test-snapd-with-default-configure${SUFFIX}"
```

Comment:
it is automatically removed during the generic cleanup, you don't need to remove the snap in the test

Reply:
Thanks, removed this step